### PR TITLE
Add ChemPy [ready for review]

### PR DESCRIPTION
**Pull request by bjodah**
This builds upon gh-12678, it bumps the version of one of the requirements. Hopefully it also regenerates CI logs which are no longer available in the referenced pull request.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bjodah** [comment]
cc @ProfLeao

**Comment by ProfLeao** [comment]
> cc @ProfLeao

Thanks for the mention. Any tasks still pending for the inclusion of version 0.8.1?

**Comment by bjodah** [comment]
I don't think so, I just wanted to see if you could think of anything. If not I think this is ready. So I guess we'll simply wait for someone to find the time to review. Thank you for getting started with this and being willing to help maintain the recipes!.

**Comment by ProfLeao** [comment]
> I don't think so, I just wanted to see if you could think of anything. If not I think this is ready. So I guess we'll simply wait for someone to find the time to review. Thank you for getting started with this and being willing to help maintain the recipes!.

Chempy is very important to my work, it’s a pleasure to do that. I'm sorry I didn't dedicate myself more recently, but the work has taken me a long time too.

File: recipes/chempy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - python >=3.7
+  run:
+    - matplotlib-base >=2.2.5
+    - notebook >=5.7.8
+    - numpy >=1.16.4
+    - python >=3.7
+    - scipy >=1.2.3
+    - sym >=0.3.4
+    - sympy >=1.5.1
+    - pytest
```
Comment (chrisburr) [inline recipes/chempy/meta.yaml:30]: Is pytest really a runtime dependency? It's fine if it is, just unusual

Comment (bjodah) [reply recipes/chempy/meta.yaml:30]: It is, perhaps bad design to do a module level import of it in a util module, but here's the import:
https://github.com/bjodah/chempy/blob/ea9b8766f26cecd363ef3293b0c4d0c6ae357063/chempy/util/testing.py#L8

**Comment by bjodah** [comment]
@chrisburr thank you for the review!

**Pull request merged**